packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -97,22 +103,29 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
                 if (!resource) {
                     throw new Error(`${VscodeCommands.OPEN.id} command requires at least URI argument.`);
                 }
+
                 if (!URI.isUri(resource)) {
                     throw new Error(`Invalid argument for ${VscodeCommands.OPEN.id} command with URI argument. Found ${resource}`);
                 }
 
-                let options: TextDocumentShowOptions | undefined;
-                if (typeof columnOrOptions === 'number') {
-                    options = {
-                        viewColumn: columnOrOptions
-                    };
-                } else if (columnOrOptions) {
-                    options = {
-                        ...columnOrOptions
-                    };
+                if (await this.shouldCreateBeforeOpen(resource)) {
+                    const uri = new TheiaURI(resource);
+                    const msg = `Unable to open "${uri}": <p>File not found!`;
+                    this.messages.showMessage({ type: MessageType.Error, text: msg, actions: ['CREATE'] })
+                        .then(res => {
+                            if (res === 'CREATE') {
+                                this.getOrCreateDirectory(uri.parent).then(parent => {
+                                    if (parent) {
+                                        this.fileSystem.createFile(resource.toString()).then(async () => {
```

Comment:
What if you try to create this in a place where you don't have permission? Do you know what VS code does in that case?

Reply:
For me it prints an error to console:
```
root ERROR Request createFolder failed with error: EACCES: permission denied, mkdir '/home/jeremy1' Error: EACCES: permission denied, mkdir '/home/jeremy1'
```
No error messages in Theia IDE

Reply:
In case te required file cannot be created vscode shows an error message like: `Unable to write file '/home/wrongUser/projects/project/aFile.ts'`. So, I've added a similar error processing.